test/test_subdir_package.py

```diff
@@ -51,13 +51,13 @@ def test(capfd, tmp_path):
         add_env={
             "CIBW_BEFORE_BUILD": "python {project}/bin/before_build.py",
             "CIBW_TEST_COMMAND": "python {package}/test/run_tests.py",
-            # this shouldn't depend on the version of python, so build only CPython 3.6
-            "CIBW_BUILD": "cp36-*",
+            # this shouldn't depend on the version of python, so build only CPython 3.8
+            "CIBW_BUILD": "cp38-*",
```

Comment:
```suggestion
            # this shouldn't depend on the version of python, so build only CPython 3.10
            "CIBW_BUILD": "cp310-*",
```
If bumping this, maybe it's worth bumping to 3.10 to further delay the time when it needs bumping again ?